src/spac/utils.py

```diff
@@ -365,6 +365,55 @@ def check_feature(
         )
 
 
+def check_label(

```

Comment:
this functionality is already in spac. see the function "check_annotation" on line 263 of this file (src/spac/utils.py).